src/components/HTMLEngineProvider/BaseHTMLEngineProvider.js

```diff
@@ -37,223 +22,6 @@ const defaultProps = {
     children: null,
 };
 
-const MAX_IMG_DIMENSIONS = 512;
-
-const EXTRA_FONTS = [
```

Comment:
Note: these are just all our fonts as an array instead of an object. So instead of having them duplicated here, we now pass `_.values(fontFamily)` instead to the `systemFonts` prop instead.